README.md

```diff
@@ -593,29 +593,28 @@ let noReturnClosure = { (arg) in doSomething(with: arg) } // void return type, o
 
 ###### Not this:
 ```Swift
-let noArgNoReturnClosure = { (Void) -> Void in doSomething() } // don't do this
-let noArgClosure = { (Void) -> Int in return getValue() } // don't do this
-let noReturnClosure = { (arg) -> Void in doSomething(with: arg) } // don't do this
+let noArgNoReturnClosure = { (Void) -> Void in doSomething() }
+let noArgClosure = { (Void) -> Int in return getValue() }
+let noReturnClosure = { (arg) -> Void in doSomething(with: arg) }
 ```
+**_Rationale:_** A `Void` return type can be inferred, thus it is unnecessarily verbose to include it
 
-However, when defining closure type, `Void` is preferred to `()`
+When defining closure type, prefer `()` for parameters, and `Void` for return types.
```

Comment:
The SwiftLint issue relating to that rule linked to a post saying that it matches Apple APIs, and that `Void` is clearer to read than `()`
http://ericasadun.com/2015/05/11/swift-vs-void/